lms/djangoapps/mobile_api/video_outlines/serializers.py

```diff
@@ -163,19 +166,32 @@ def video_summary(course, course_id, video_descriptor, request, local_cache):
     """
     returns summary dict for the given video module
     """
-    # First try to check VAL for the URLs we want.
-    val_video_info = local_cache['course_videos'].get(video_descriptor.edx_video_id, {})
-    if val_video_info:
-        video_url = val_video_info['url']
+    # Get encoded videos
+    encoded_videos = local_cache['course_videos'].get(video_descriptor.edx_video_id, {})
```

Comment:
We should use a different cache key, because we are caching a different kind of object.